Updating getting_started.md File (#144)
VowpalWabbit/vowpalwabbit.github.io: _tutorials/getting_started.md

This diagnostic ouput shows that the number of bits from the hash function is 18 (more than enough for this example).

> Use `-b <Number of bits>` to adjust the number of bits to be used from the hash function. 

Reviewer: ```suggestion
Use `-b <number of bits>` to adjust the number of bits to be used from the hash function. 
```

> So you have `vw -b <Number of bits> house_dataset` to run the code again

Reviewer: ```suggestion
For example: `vw -b 10 house_dataset`
```

If the data is noisy, you need a larger data-set or multiple passes to predict well. For massive data-sets, the learning rate decays towards `0` by default.

Use  `-l new_learning_rate` to adjust the learning rate up or down.

> So you can try something like `vw -l 0.4` if you want the new learning rate to be 0.4. You can always choose the learning rate based on your project

Reviewer: ```suggestion
Use `-l <learning rate>` to adjust the learning rate up or down.

For example: `vw -l 0.4`
```

initial_t = 0
```

> >**Note:** Learning rates often decay over time, and this diagnostic output specifies the initial time. You can adjust with `--initial_t new_time`, although this is rarely necessary these days.

Reviewer: ```suggestion
>**Note:** Learning rates often decay over time, and this diagnostic output specifies the initial time. You can adjust with `--initial_t <time>`, although this is rarely necessary these days.
```

> For example: `vw --initial_t 4`

Reviewer: This should be in the quote

> So you can have something like `vw --initial_t 4` if you want the new initial time to be 4

Reviewer: ```suggestion
For example: `vw --initial_t 4`
```

Use `--cache_file housing.cache` to override the default cache file name.

> So you have something like `vw --cache_file housing.cache house_dataset` if you want to use the cache and not the original file

Reviewer: This example is misleading because house_dataset is implicitly the -d value here. Additionally if housing.cache does not exist it will generate it. I think line 177 summarizes what goes on here well enough.

Author: I agree

The cache file is created the first time you use `-c`. If the cache exists and is newer than the dataset, that file is used by default.

Use `-c` for multiple passes `--passes`, so Vowpal Wabbit caches the data in a faster format (passes > 1 should be much faster).  If you want to experiment with the same dataset over and over, it is highly recommended to pass `-c` every time you train.

> So we can have something like: ```vw -c --passes 50 house_dataset```

Reviewer: ```suggestion
For example: `vw -c --passes 50 -d house_dataset`
```
